protocol/src/test/java/io/camunda/zeebe/protocol/ZbColumnFamiliesTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.Arguments;
+
+class ZbColumnFamiliesTest {
+
+  @Test
+  void shouldNotReuseEnumValues() {
```

Comment:
Might make also sense to test that they increasing by one abd there are no gaps?. Or check against the ordinal that they have the same value. 

This could be then also backported I guess so we make sure that we don't miss them